tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -579,6 +579,15 @@ def test_model_monitoring_with_regression(self):
         # Deploy the serving function
         serving_fn.deploy()
 
+        # Validate that the model monitoring batch access key is replaced with an internal secret
+        batch_function = mlrun.get_run_db().get_function(
+            name="model-monitoring-batch", project=self.project_name
+        )
+        assert (
+            "mlrun-auth-secrets"
```

Comment:
Please don't use hard-coded strings here, but rather use `mlrun.model.Credentials.secret_reference_prefix` (which should be `$ref`) - it should be included in the access-key reference in the function.
Also, you should use `mlconf.secret_stores.kubernetes.auth_secret_name.format(hashed_access_key="")` instead of the hard-coded `mlrun-auth-secrets`